assets/playground.html

```diff
@@ -816,6 +833,20 @@
           this.ask();
         },
 
+        /**
+         * @param {PointerEvent} event
+         */
+        handleTTSMessage(event) {
+          if (!!window.speechSynthesis) {
+            if (window.speechSynthesis.speaking || window.speechSynthesis.pending) {
+              window.speechSynthesis.cancel();
+            } else {
+              let messageToUtter = event.target.parentElement.parentElement.parentElement.innerText;
```

Comment:
> let messageToUtter = event.target.parentElement.parentElement.parentElement.innerText;

This is not elegant. Even small HTML changes may cause errors, and it's difficult to find the problems.

Why not make `handleTTSMessage` accept content as a parameter, like `handleCopyMessage`?

https://github.com/sigoden/aichat/blob/80d3edbd43e874bb30decf1c5d13834a9a896007/assets/playground.html#L605

Reply:
great idea, I'll do it now